code/controllers/master.dm

```diff
@@ -289,34 +308,33 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 	var/list/tickersubsystems = list()
 	var/list/runlevel_sorted_subsystems = list(list()) //ensure we always have at least one runlevel
 	var/timer = world.time
-	for (var/thing in subsystems)
-		var/datum/controller/subsystem/SS = thing
-		if (SS.flags & SS_NO_FIRE)
+	for (var/datum/controller/subsystem/subsystem as anything in subsystems)
+		if (subsystem.flags & SS_NO_FIRE)
 			continue
-		SS.queued_time = 0
-		SS.queue_next = null
-		SS.queue_prev = null
-		SS.state = SS_IDLE
-		if (SS.flags & SS_TICKER)
-			tickersubsystems += SS
+		subsystem.queued_time = 0
+		subsystem.queue_next = null
```

Comment:
You have been docked 420 social credit score points.

Reply:
![image](https://user-images.githubusercontent.com/15794172/145674214-a74cbbb9-237b-476e-b7fc-c5a46790ac9b.png)